src/DataService/Annotation/DataService.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace Lullabot\Mpx\DataService\Annotation;
+
+/**
+ * @Annotation
+ *
+ * Each MPX data service exposes one or more objects. We require annotations
+ * on object implementations so calling code can discover what services are
+ * currently implemented. In general, there should only be one implementation
+ * of a given (path, service, schema) triple.
+ *
+ * @todo Use symfony validation to assert requirements.
+ * @todo Mark the required values with @Required
+ * @todo Rename to DataServiceObject
+ * @todo Can we infer /data/ from path and assume it's always consistent?
+ *
+ * @see http://docs.doctrine-project.org/projects/doctrine-common/en/latest/reference/annotations.html
```

Comment:
Thanks for this link! This is just what I need to read right now to better understand all this approach, which sounds nice so far by the way.